src/main/java/uk/gov/hmcts/reform/pip/account/management/service/PublicationService.java

```diff
@@ -50,4 +52,21 @@ public String sendNotificationEmail(String emailAddress, String forename, String
             return "Email request failed to send: " + emailAddress;
         }
     }
+
+    /**
+     * Method which sends a request to publication services to email the P&I team.
+     * With a list of all media applications in the database.
+     *
+     * @return String for logging success or failure
+     */
+    public String sendMediaApplicationReportingEmail(List<MediaApplication> mediaApplicationList) {
```

Comment:
(Not technically caused by this PR however it will cause this PR to not work in environments)

We'll need to add the auth to talk to publication services for this, as currently it won't provide auth / retrieve 401